### PR TITLE
New Pipeline Integration

### PR DESCRIPTION
**Pull request by justinorringer**
## What?
Adding payload-tracker-go to the new pipeline

## Why?
Consider what business or engineering goal does this PR achieves.

## How?
Describe how the change is implemented. Any noteable new libaries, APIs, or features.

## Testing
Did you add any tests for the change?

## Anything Else?
Any other notes about the PR that would be useful for the reviewer. 

## Secure Coding Practices Checklist Link
- https://github.com/RedHatInsights/secure-coding-checklist

## Secure Coding Checklist
- [ ] Input Validation
- [ ] Output Encoding
- [ ] Authentication and Password Management
- [ ] Session Management
- [ ] Access Control
- [ ] Cryptographic Practices
- [ ] Error Handling and Logging
- [ ] Data Protection
- [ ] Communication Security
- [ ] System Configuration
- [ ] Database Security
- [ ] File Management
- [ ] Memory Management
- [ ] General Coding Practices

### CODE REVIEW

**Review by tahmidefaz — APPROVED** [review]

**Comment by tahmidefaz** [comment]
@justinorringer any idea why the pipeline is failing?

File: unit_test.sh [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+
+echo -e "\n---------------------------------------------------------------\n"
+
+echo "Pulling postgres image"
+docker pull $POSTGRES_IMAGE
+
+echo -e "\n---------------------------------------------------------------\n"
+
+echo "Starting postgres container"
+docker run --health-cmd pg_isready --health-interval 10s --health-timeout 5s --health-retries 5 -d --rm --name postgres -e POSTGRES_PASSWORD=crc -e POSTGRES_USER=crc -e POSTGRES_DB=crc -h 0.0.0.0:5432:5432 $POSTGRES_IMAGE
```
Comment (dehort) [inline unit_test.sh:44]: I think the env var names need to adjusted.
`
 -e POSTGRESQL_PASSWORD=crc -e POSTGRESQL_USER=crc -e POSTGRESQL_DATABASE=crc
`
I think this is causing the image/container to fail to start and that's causing the migration to fail to connect.

I'm running into some issues with podman locally...so I'm struggling to get this working locally but I think we need to start with this change.

File: unit_test.sh [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+
+echo -e "\n---------------------------------------------------------------\n"
+
+echo "Running container"
+docker run -d --rm --name "$TEST_CONTAINER_NAME" "$IMAGE" sleep infinity
+
+echo -e "\n---------------------------------------------------------------\n"
+
+echo "Installing dependencies"
+docker exec --workdir /workdir "$TEST_CONTAINER_NAME" make install > 'artifacts/install_logs.txt'
```
Comment (dehort) [inline unit_test.sh:60]: I don't think this is necessary.  I think the `build` step will install the dependencies.

File: unit_test.sh [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+
+echo -e "\n---------------------------------------------------------------\n"
+
+echo "Building migrations"
+docker exec --workdir /workdir "$TEST_CONTAINER_NAME" make pt-migration -B > 'artifacts/build_logs.txt'
```
Comment (dehort) [inline unit_test.sh:65]: I don't think this is necessary.  I think the `make run-migration` step will build the `pt-migration` binary.

**Review by dehort — COMMENTED** [review]
Looking good!

**Comment by justinorringer** [comment]
Not using this, closing.

**Pull request closed without merging**